modules/ROOT/pages/_partials/proc_self-signed-certs.adoc

```diff
@@ -51,6 +51,11 @@ Drive which is accessible from the device, browse to the certificate and
 proceed to download it.
 * USB - Attach the device to the machine hosting
 OpenShift via USB and drag the certificate to a devices file system. Here is an https://support.google.com/nexus/answer/2840804?hl=en[example guide for Google Nexus devices]. It may be different for other devices.
+* Android Debug Bridge (adb) - Use the adb push command to push the certificate to device or emulator:
```

Comment:
this looks like a good change, but does it open a can of worms.. how do i use adb, why didn't you mention it earlier, do I need to use adb? etc

What would work is if we had a better link above to a proper google article, that would make this 'adb' bit seem like a natural next step

Reply:
Good point, as we have already called out in the documentation about setting up your environment, a developer will get adb out of the box when installing android studio.
Would making Android Debug Bridge a clickable link to some google documentation be sufficient?

Reply:
@ciaranRoche  yeah, go for it